addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -119,7 +121,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         if (cspHeaderFound) {
             List<PolicyError> observedErrors = new ArrayList<>();
             PolicyErrorConsumer consumer =
-                    (severity, message, directiveIndex, valueIndex) -> {
+                    (Severity severity, String message, int directiveIndex, int valueIndex) -> {
```

Comment:
Why?

Reply:
Sonarlint...I can revert if you hate it

Reply:
Not hate (well, maybe a little for drive-by changes hehe), but unless required for disambiguation we are not adding the types to any lambdas, IMO better go for consistency (I don't really mind either way as long as we apply the rules consistently).

Reply:
That's fine, I'll remove and disable the rule for now.